lib/board.rb

```diff
@@ -0,0 +1,45 @@
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    reset!
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+  def display
+    puts " #{@cells[0..2].join(" | ")} "
+    puts "-----------"
+    puts " #{@cells[3..5].join(" | ")} "
+    puts "-----------"
+    puts " #{@cells[6..8].join(" | ")} "
+  end
+
+  def position(index)
+    @cells[index.to_i - 1]
+  end
+
+  def full?
+    @cells.include?(" ") ? false : true
```

Comment:
Avoid using the ternary operator to return `true` and `false`. Instead, rely on the implicit boolean return value of the expression. You can reverse (or negate) the boolean with the `!` (bang) operator.

```
def full?
  !@cells.include?(" ")
end
```